test/server/lib/DocApi.ts

```diff
@@ -4434,11 +4434,28 @@ function testDocApi() {
           ['ModifyColumn', 'Table1', 'B', { type: 'Bool' }],
         ], chimpy);
 
-        const webhook = await autoSubscribe('200', docId, {
+        const modifyColumnNotInColumnIds = async (newValues: { [key: string]: any; } ) => {
+          await axios.post(`${serverUrl}/api/docs/${docId}/apply`, [
+            ['UpdateRecord', 'Table1', newRowIds[0], newValues],
+          ], chimpy);
+          await delay(100);
+          assert.isFalse(successCalled.called());
+          successCalled.reset();
+        };
+        const modifyColumnInColumnIds = async (newValues: { [key: string]: any; }) => {
+          await axios.post(`${serverUrl}/api/docs/${docId}/apply`, [
+            ['UpdateRecord', 'Table1', newRowIds[0], newValues],
+          ], chimpy);
+          await delay(100);
+          assert.isTrue(successCalled.called());
+          await successCalled.waitAndReset();
+        };
```

Comment:
Nitpicking: Here is something I would suggest, in order to separate operations to expectations:

```suggestion
        const modifyColumn = async (newValues: { [key: string]: any; } ) => {
          await axios.post(`${serverUrl}/api/docs/${docId}/apply`, [
            ['UpdateRecord', 'Table1', newRowIds[0], newValues],
          ], chimpy);
          await delay(100);
        };
        const assertSuccessNotCalled = async () => {
          assert.isFalse(successCalled.called());
          successCalled.reset();
        };
        const assertSuccessCalled = async () => {
          assert.isTrue(successCalled.called());
          await successCalled.waitAndReset();
        };
```

This way, the tests look like this, and I would say your expectations (assertions) would be clearer:
```ts
        await modifyColumn({ C: 'c2' });
        await assertSuccessNotCalled();
        await modifyColumn({ A: 19 });
        await assertSuccessCalled();
```

What do you think?